### DISCUSSION TITLE
how to open the original picture?

### DISCUSSION THREAD

**Discussion by foxfire881** [photoprism/photoprism#2800] [General]
my digital picture is 6016pix * 4016pix (filesize: 24M, dpi: 240) , but when I open it in photoprism or download it after indexed, it become 1797pix * 1200pix (filesize: 2.2M, dpi: 96), although I have set all the options  max.

so, how should I disable downscaling? I want to browse original pictures.

thanks~

![99](https://user-images.githubusercontent.com/273961/197826976-32656bb5-da4d-4f2f-bfa7-3dc02d5fabb1.PNG)

**Comment by lastzero** [comment]
You want to look at it with maximum zoom all the time or do you want your web browser, who is not built for that, to scale the image down in bad quality? See our advanced settings docs for a little background information as to why thumbs are generally a good idea. Of course it is also possible to download the original file, maybe depending on what specific button or menu you are referring to. I've done it many times today.

**Reply by foxfire881** [reply]
Thanks for relpy.

Have you ever used Adobe Lightroom Classic? It also indexed original pictures, but it could smoothly zoom in/out original pictures, and the quality is very well with every zoom level. I hope photoprism could work like this, too.

On the other hand，after photoprism indexed original photos, it seems that it will create two thumbs , one is the cover, the other is the lower quality after downscaling, it cost too much space of my harddisk if I have many original photos. In fact, I only need one thumb for cover.

**Reply by graciousgrey** [reply]
Our current zooming functionality is limited by the photoviewer we use. 
Improvements are planned: https://github.com/photoprism/photoprism/issues/1307.

Thumbnail generation can be configured, for more information see:  https://docs.photoprism.app/user-guide/settings/advanced/#images

**Reply by lastzero** [reply]
Kindly note that Lightroom Classic is NOT a web app. We have to work with what the average browser provides. Besides, Lightroom has many other shortcomings. Nobody is asking us to add them. As for new features, our users decide what budget we have and how fast we can develop.
